signalr-node-server/signalr.js

```diff
@@ -391,12 +388,15 @@ class Hub {
         switch (message.type) {
             case signalr.MessageType.Invocation:
                 try {
-                    var method = this._methods[message.target.toLowerCase()];
```

Comment:
Removed `.toLowerCase()` since the guideline indicates the target is case sensitive https://github.com/aspnet/AspNetCore/blob/master/src/SignalR/docs/specs/HubProtocol.md

Reply:
That's not correct. The implementations are all case insensitive.

Reply:
Well, I had to listen for `on("sendmessage")` instead of `on("SendMessage")` because of this (took me a while to figure that out and I think .NET matches it automatically but in NodeJS it isn't afaik) and it doesn't feel right here. What do you advise?

Reply:
To revert it.